src/test/mocks/BeaconChainMock.t.sol

```diff
@@ -176,7 +176,7 @@ contract BeaconChainMock is Logger {
     /// - Setting their current/effective balance
     /// - Assigning them a new, unique index
     function newValidator(bytes memory withdrawalCreds) public payable returns (uint40) {
-        print.method("newValidator");
+        console.log("BeaconChain.newValidator()");
```

Comment:
Why get rid of Logger?

Feels like we print inconsistently in a lot of places,,